src/main/java/Duke.java

```diff
@@ -1,10 +1,30 @@
+import java.util.Scanner;
 public class Duke {
+
+    public static void greet() {
+        System.out.println("________________________________");
+        System.out.println("Hello! I'm ChatTPG");
+        System.out.println("What can I do for you?");
+        System.out.println("________________________________");
+    }
+
+    public static void bye(){
```

Comment:
Perhaps include a space between `bye()` and the opening curly brace to maintain consistency throughout your code.

Reply:
noted, will change and commit, thanks!